lib/IRGen/GenProto.cpp

```diff
@@ -1832,10 +1832,6 @@ static llvm::Constant *emitResilientWitnessTable(IRGenModule &IGM,
     SILFunction *Func = entry.getMethodWitness().Witness;
     llvm::Constant *witness;
     if (Func) {
-      // Force the thunk to be emitted in the current translation unit
-      // when in multi-threaded mode.
-      IGM.IRGen.forceLocalEmitOfLazyFunction(Func);
```

Comment:
I think this was the only usage of forceLocalEmitOfLazyFunction so you can remove it

Reply:
I now use it in ensureRelativeSymbolCollocation.